game/core/models/base_player_inventory.py

```diff
@@ -88,6 +88,11 @@ def equip_item(self, item_name: str) -> Sprite:
         self.equipped_item = self.inventory[item_name]
         return self.equipped_item.sprite
 
+    @beartype
+    def unequip_item(self) -> None:
+        """Unequip current item."""
+        self.equipped_item = None
```

Comment:
We should probably use `store_equipped_item` instead

https://github.com/DesignPatternsAdventure/game/blob/ddd7f1ba38fd029a335146ed2269c20bc15321fb/game/core/models/base_player_inventory.py#L79-L83

Reply:
Ah okay! I deleted the new method and renamed `store_equipped_item` to `unequip_item` to be more straightforward. Let me know if there are any concerns, I can revert it back easily.